Menus/RadialMenu/RadialMenu.cs

```diff
@@ -83,6 +83,8 @@ public bool visible
 
 		public Node? node { get; set; }
 
+		public float hideDistance {get { return Mathf.Infinity; } }
```

Comment:
I would probably leans towards never hide since would technically be doing less but we should probably set up some kind of test scene where a user is surrounded by UI and see how both feel.